src/components/DocumentSelector/index.js

```diff
@@ -101,24 +102,24 @@ class DocumentSelector extends Component<Props & WithDefaultOptions> {
 
 const LocalisedDocumentSelector = localised(DocumentSelector)
 
-const withDefaultOptions = (types: Object) => (props: Props) => (
+const withDefaultOptions = (iconCopyDisplayByType: Object) => (
+  props: Props
+) => (
   <LocalisedDocumentSelector
     {...props}
     defaultOptions={() => {
-      const typeList = Object.keys(types)
+      const typeList = Object.keys(iconCopyDisplayByType)
       const group = props.group
-      return typeList.map((value) => {
+      return typeList.map((type) => {
         const {
-          icon = `icon-${kebabCase(value)}`,
+          icon = `icon-${kebabCase(type)}`,
           hint,
           warning,
-          ...other
```

Comment:
Can't tell what this was for but guessing it wasn't needed anymore?

Reply:
Same. I couldn't work out why it was there but at the same time the component it's being passed into is not using it or passing it onward in any way so I removed it.